#goal
---
#result - the centers of the balls we're detecting
geometry_msgs/PoseArray centers
---
#feedback
